cli/src/runner.rs

```diff
@@ -6,10 +6,10 @@ use api;
 use context::{junit::parser::JunitParser, repo::BundleRepo};
 
 use crate::api_client::ApiClient;
-use crate::codeowners::CodeOwners;
-use crate::constants::{EXIT_FAILURE, EXIT_SUCCESS};
-use crate::scanner::{FileSet, FileSetCounter};
-use crate::types::{QuarantineBulkTestStatus, QuarantineRunResult, RunResult, Test};
+use bundle::{FileSet, FileSetCounter};
+use bundle::{QuarantineBulkTestStatus, QuarantineRunResult, RunResult, Test};
+use codeowners::CodeOwners;
+use constants::{EXIT_FAILURE, EXIT_SUCCESS};
```

Comment:
Not sure why these weren't grouped before, but we should probably make them more consistent